src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -4471,6 +4486,13 @@ namespace SystemAvailabilityManager {
 				ErrorsFound = true;
 			}
 
+			if ( !lNumericFieldBlanks( 8 ) ) {
```

Comment:
@mjwitte These two fields are last one. If I extend min-fields, it will cover every field. Some fields may not be needed in certain conditions. Since the default value for this variable in the code is zero, it does not need IP to handle. I suggest to leave it as is. We can change is later.